### DISCUSSION TITLE
cant see scripts

### DISCUSSION THREAD

**Discussion by wareriley11** [Vdauphin/HeartsAndMinds#1351] [Question and answer]
having an issue where i have the option to interact with side mission and civilians but i cant see any of the words. it  just shows up as blank option. other than that it seems like everyhting is running smoothly.

![ArmA 3 Screenshot 2022 04 14 - 09 10 11 99](https://user-images.githubusercontent.com/103661073/163419766-5214d60f-a234-4844-9945-a808b842dd0f.png)
![ArmA 3 Screenshot 2022 04 14 - 09 10 15 31](https://user-images.githubusercontent.com/103661073/163419770-e0097465-1ed6-4b6f-8a06-d4866eaa3ab6.png)
![ArmA 3 Screenshot 2022 04 14 - 09 10 08 56](https://user-images.githubusercontent.com/103661073/163419772-e18f8f47-1d1b-4057-87ac-a01e929c0562.png)

**Comment by Vdauphin** [comment]
Hello,

May be, search in your .rpt for errors and fix them

Cheers

**Comment by wareriley11** [comment]
getting alot of the 

STR_BTC_HAM_PARAM_ not found errors.

**Reply by Vdauphin** [reply]
Revert each change you made until it works again
